core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1307,7 +1307,8 @@ private[spark] object Utils extends Logging {
   }
 
   /** Default filtering function for finding call sites using `getCallSite`. */
-  private def sparkInternalExclusionFunction(className: String): Boolean = {
+  private def sparkInternalExclusionFunction(className: String):
+    Boolean = {
```

Comment:
Do we really need a indentation here and https://github.com/apache/spark/pull/11301/files#diff-d239aee594001f8391676e1047a0381eR1335 ?